domains/c9pz.json

```diff
@@ -0,0 +1,13 @@
+{
+        "owner": {
+           "username": "max96git",
+           "email": "",
+           "discord": "1201832120796840036",
+           "OWL": "eyJlbmMiOiJBMTI4Q0JDLUhTMjU2IiwiYWxnIjoiUlNBLU9BRVAiLCJraWQiOiJaa1VsRmRqVThiUEstLXVVM2JJR09PVHFYYVFFS1ZINFVXOW53MTR6WTJnIn0.IKsOl5JKWlh8-8Vd2HCoJoZEeZpPKQQILsHGIrX8pChZO9X_jxL4VeuXIIF1LOAttMLu37XOsI9OB1basu4AkpPtKkl6hYZWTWoyNbOG5Ljwg9obbM_itF7lsz9iBdwR9qntPrBalHOS1TTNKhGteKnSEYQC1Ab5OC2t33tqx9JunOD2WjvvMXvUS4M0nroEyFHKmVTM1LCX0d3lYhZhQVCK8cf1AdJmQBByzU_8K2hMc_Iq-jRE6yNepPG0LOBmvtldyAYpMkeDfnJbhp439bdlPtWG0c_Gjq9zcjNG4Rorgh72O3I7LFlCBZ1hEUxidzesaMUFvFYgC5IrlXDWxA.6IWaRnC6sU5tPFd7KeBv0A.9d-t733YyTln5YPN37A7lup2pP23hYNkjVS1P4ftqMR46UQvJIbjH_wsJJlojyfy0HbnCXBhTtsW-d-kGCYXIg.08ropVbtKcvhWbRb8Enk_Q"
+        },
+    
+        "record": {
+            "TXT": "openai-domain-verification=dv-21hujylb9vaaow77qie1gl9z"
```

Comment:
TXT records alone doesn't do anything.

Reply:
Do i add something else with the TXT record?

Reply:
Yes, since TXT records are normally to verify ownership of the domain. Like A records and TXT verification, or AAAA records and TXT, etc.

 I've also never seen an OpenAI verification record.

Reply:
Alright